src/sidecar/sidecarManager.ts

```diff
@@ -104,11 +104,10 @@ export class SidecarManager {
       this.startTailingSidecarLogs();
     }
 
-    for (let i = 0; i < 10; i++) {
-      // Get our current auth header out of the secret store
-      // (If it's not there, will return empty string, and we'll end up down either path 2. or 3. below)
+    const max_attempts = 20;
```

Comment:
Could this be a module-level const to avoid being passed into `this.startSidecar()`?

Reply:
Or static property on the manager class, for that matter